### ISSUE TITLE
Toolbar buttons are not read by screen reader correctly

### ISSUE THREAD

**Issue by Zeddox** [xdan/jodit#725]
Using [NVDA](https://www.nvaccess.org/) screen reader, the toolbar buttons are not being reported correctly. For example, the Bold button reads: "Toggle Button, Not Pressed"

**Jodit Version:**  3.4.29
**NVDA Version:** 2021.2

**Expected behavior:**
Toolbar buttons are read with their labels, e.g. "Bold, Not Pressed"

**Actual behavior:**
Toolbar buttons are read without labels, leading to a user not knowing what toolbar action is focused.

_Note:_ It looks like the aria-label attribute is set on a span that contains the toolbar button. Here are a few links that may be useful:
[Short note on aria-label](https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/)
[NVDA github issue on reading <span> elements](https://github.com/nvaccess/nvda/issues/7807)

**Comment by xdan** [comment]
@Zeddox Try in 3.8.2. Should be fixed

**Comment by Zeddox** [comment]
Thanks, confirmed the fix today.

**Comment by cgalpin** [comment]
This fix seems to have gotten lost along the way - I don't see this in the current version. Buttons are being read as  "Toggle Button, Not Pressed" again.
